README.md

```diff
@@ -144,12 +161,91 @@ Include content from "Intro" up to the next heading of the same level:
 `inline`
 :   Flag that tells the preprocessor to replace sequences of whitespace characters of many kinds (including `\r`, `\n`, and `\t`) with single spaces (` `) in the included content, and then to strip leading and trailing spaces. It may be useful in single-line table cells. Default value is `false`.
 
+`project_root`
+:   Path to the top-level (“root”) directory of Foliant project that the included file belongs to. This option can be necessary to resolve correctly the `!path` and the `!project_path` modifiers in the included content.
+
+    >    **Note**
+    >
+    >    By default, if a local file is included, `project_root` points to the top-level directory of the current Foliant project, and if a file in a remote Git repository is referenced,.`project_root` points to the top-level directory of this repository. In most cases you do not need to override the default behavior.
```

Comment:
>    By default, if a local file is included, `project_root` points to the top-level directory of the current Foliant project, and if a file in a remote Git repository is referenced, `project_root` points to the top-level directory of this repository. In most cases, you don't need to override the default behavior.